iep-007/README.adoc

```diff
@@ -0,0 +1,153 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+= IEP-007: Kubernetes cluster upgrade
+
+:toc:
+
+.Metadata
+[cols="2"]
+|===
+| IEP
+| 007
+
+| Title
+| Kubernetes cluster upgrade 
+
+| Author
+| link:https://github.com/olblak[Olblak]
+
+| Status
+| :speech_balloon: In-process
+
+| Type
+| [Process]
+
+| Created
+| 2016-07-04
+|===
+
+
+
+== Abstract
+
+As Kubernetes project often releases new versions, it's interesting to enjoy new features or bugfixes introduced by those new versions. 
+Therefor we need an easy way to upgrade Kubernetes cluster on a regular basis.
+
+
+== Specification
+
+Currently there is two main strategies to upgrade a cluster. +
+
+. Upgrading an existing cluster.
+. Migration on a second cluster.
+
+As Azure do not (yet) provide any tools to upgrade existing clusters, we have to upgrade them manually.  +
+It appears to be easier and safer to deploy a new cluster and re-deploy all resources on the new one.
+
+Until the new cluster stays in the same region than the previous one, we can keep persistent data from blob storage and attach them to the new cluster. +
```

Comment:
> Until the new cluster stays in the same region than the previous one,

I think this is incorrectly written. I believe you mean to say "As long as the new cluster stays in the same region" right? As long as we stay in East US 2, we can use the same blob storage accounts?